### PR TITLE
New context menu option to randomize portraits for selected personnel.

### PR DESCRIPTION
**Pull request by Qwertronix**
^title, in Personnel tab. 

Does not check if the person already had a portrait; if it did, it simply runs the generator anyway which will override the previous portrait. 

As-is this works for me, but we could also add the option of having it only generate portraits for selected personnel who didn't have one.

### CODE REVIEW

**Comment by Qwertronix** [comment]
Per feedback on slack, making it ignore personnel who already have a portrait by default.

**Comment by Qwertronix** [comment]
Oh, if accepted this should also resolve #321.

File: MekHQ/src/mekhq/resources/PersonnelTableMouseAdapter.properties [hunk 1]
```diff
@@ -21,6 +21,7 @@ changeCallsign.text=Change Callsign...
 changeMDClass.text=Change Manei Domini Class
 changeMDRank.text=Change Manei Domini Rank
 changePortrait.text=Change Portrait...
+randomizePortrait.text=Randomize Portrait...
```
Comment (NickAragua) [inline MekHQ/src/mekhq/resources/PersonnelTableMouseAdapter.properties]: Usually, we use "..." to indicate that there's going to be some kind of popup. So, I think we should get rid of the ...

**Comment by Qwertronix** [comment]
Doh! Will do.

**Comment by Qwertronix** [comment]
@NickAragua Done.

**Pull request merged**